tests/requirements/test/test_hal_scb.rs

```diff
@@ -0,0 +1,25 @@
+// Test scenario:
+// - Verify that icache can be disabled/enabled/cleared
+// - Verify that dcache can be disabled/enabled/cleared
+
+/// @SRS{ROS-FUN-BSP-SCB-020}
+/// @SRS{ROS-FUN-BSP-SCB-030}
```

Comment:
It's not covered for now, right?

Reply:
yes, 030 is not covered, removed and added TODO for that